<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="10" column="1">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QCheckBox" name="linesCheckBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Линии</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="pointsCheckBox">
        <property name="text">
         <string>Точки</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Вращения модели</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>14</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="xRotateSpinBox">
          <property name="minimum">
           <number>-360</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="singleStep">
           <number>15</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>14</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="yRotateSpinBox">
          <property name="minimum">
           <number>-360</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="singleStep">
           <number>15</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>14</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="zRotateSpinBox">
          <property name="minimum">
           <number>-360</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="singleStep">
           <number>15</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="rotateButton">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="edgesLabel">
      <property name="text">
       <string>Ребра: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="vertexesLabel">
      <property name="text">
       <string>Вершины: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QLabel" name="filePath">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="plusButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="minusButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="11">
     <widget class="OpenGLWidget" name="openGL">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Масштаб</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="scaleSpinBox">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scaleButton">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>18</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Сдвиг модели</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>14</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>14</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>14</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="xMoveSpinBox">
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="yMoveSpinBox">
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="zMoveSpinBox">
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="moveButton">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="open_file">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>OpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
